.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,22 +50,60 @@ Steps to reproduce the behavior:
 
 A clear and concise description of what you expected to happen.
 
-**Logs**
-
-Log file gives us deep insight into the behavior and performance of the extension, so we strongly
-recommend you upload it.
+### **Logs**
 
-Go to the command palette (Cmd+Shift+P on Mac, Ctrl+Shift+P on Windows), search `Open Angular Server log`
+Log file gives us deep insight into the behavior and performance of the extension. If the issue is a performance
+problem or an error occured, please provide the output of the log file below.
 
 Set Angular Log level to verbose.
-Open settings (File/Preferences/Settings on Windows, Code/Preferences/Settings on Mac), go to Extensions/Angular Language Service, set `angular.log` to `verbose`.
 
-**Screenshots**
+<pre><code>
+<!-- If the issue is accompanied by log file output, please share it below.
+
+Set the Angular Log level to verbose by opening settings (File/Preferences/Settings on Windows, Code/Preferences/Settings on Mac), go to Extensions/Angular Language Service, set `angular.log` to `verbose`.
+
+To open the log file, go to the command palette (Cmd+Shift+P on Mac, Ctrl+Shift+P on Windows), search `Open Angular Server log`.
+
+Alternatively, if the log file is log, it can be uploaded instead of pasting below. -->
```

Comment:
We should tell users where to upload the file.
https://gist.github.com/ perhaps?

Reply:
Github has native file upload support https://docs.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests#:~:text=To%20attach%20a%20file%20to,images%20directly%20into%20the%20box.